components/src/md3/ChannelValue/ChannelValue.tsx

```diff
@@ -110,7 +110,7 @@ export const ChannelValue: React.FC<ChannelValueProps> = (props) => {
     const getIcon = useCallback(() => {
         if (icon) {
             return (
-                <View style={[{ marginRight: Math.round(fontSize / 3) }]}>
+                <View style={[{ marginRight: Math.round(fontSize / 3), paddingTop: 2 }]}>
```

Comment:
I don't think this is the appropriate fix for the icon alignment with the text. If you look at the showcase vs the MD3 demo you'll see that the showcase examples are now off. Not sure why we're getting different results, but showcase is where we should be testing things now and the icon is not aligning properly now.

Reply:
The text seems to be aligned without any additional styles in the showcase